meinberlin/templates/a4dashboard/project_create_form.html

```diff
@@ -37,8 +37,6 @@ <h1 class="u-first-heading">{% trans "Create project" %}</h1>
 
                 {% include 'a4dashboard/includes/form_field.html' with field=form.is_public %}
 
-                <p>{% trans 'After saving the draft project you can further customize and edit your project and eventually publish it.' %}</p>
```

Comment:
Cool! Didn't I copy that over to all the create-thingies? Did you check? Otherwise I will!